daffodil-core/src/main/scala/org/apache/daffodil/lib/util/Misc.scala

```diff
@@ -761,4 +761,4 @@ object Misc {
     }
   }
 
-}
+}
```

Comment:
Please undo this change, files should end with a new line. This might require a change in your IDE settings.